.changeset/cuddly-boats-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/focus-lock": patch
```

Comment:
Sorry! 😢 The changeset is not needed for this PR, as it is related to refactoring. Please delete this file. @hong-nguyen13

Reply:
I've deleted it! Thank you👌